src/api/fetchRegistryKeys.ts

```diff
@@ -0,0 +1,23 @@
+// Import Third-party Dependencies
+import * as httpie from "@myunisoft/httpie";
+
+// CONSTANTS
+const kNpmRegistryEndpoint = "https://registry.npmjs.org/-/npm/v1/keys";
+export interface NpmRegistryKeysResponse {
+    keys: {
+        keyid: string;
+        keytype: string;
+        scheme: string;
+        key: string;
+        expires: string | null;
+    }[];
+}
+
+export async function fetchRegistryKeys(): Promise<NpmRegistryKeysResponse> {
+    try {
+        const { data } = await httpie.get<NpmRegistryKeysResponse>(kNpmRegistryEndpoint);
+        return data;
+    } catch (error) {
+        throw new Error(`Failed to fetch registry keys: ${(error as Error).message}`);
+    }
+}
```

Comment:
No need to catch and re-throw error and you can probably return data.keys and update the interface